Sources/apm-agent-ios/Utils/OpenTelemetryHelper.swift

```diff
@@ -56,7 +56,7 @@ public class OpenTelemetryHelper {
     }
 
     return URL(string: "\(config.collectorTLS ? "https://" : "http://")\(config.collectorHost)\( port.isEmpty ? "" : ":\(port)")")
-
+    return URL(string: "\(config.collectorTLS ? "https://" : "http://")\(config.collectorHost)\( port.isEmpty ? "" : ":\(port)")\(config.collectorPathPrefix ?? "")")
```

Comment:
This will result in the otel endpoints looking something like: 
`https://myhost.com/path-prefix/v1/trace` is this the intention?